cdap-app-fabric/src/main/java/io/cdap/cdap/security/impersonation/DefaultUGIProvider.java

```diff
@@ -172,4 +202,59 @@ private File localizeKeytab(Location keytabLocation) throws IOException {
 
     return localKeytabFile.toFile();
   }
+
+  /**
+   * Returns the runtime user supplied arguments to a program if entity is of type ProgramRun
+   *
+   * @param programId the entity id
+   * @return properties map
+   */
+  private Map<String, String> getRuntimeProperties(NamespacedEntityId programId) {
+    if (programId instanceof ProgramRunId) {
```

Comment:
It's better to flip the logic to make the code less indented, hence easier to read.

```java
if (!(programId instanceof ProgramRunId) {
  LOG.debug(....);
  return Collections.emptyMap();
}

ProgramRunId programRunId = (ProgramRunId) entityId;
...
```

Reply:
Done. Thanks